packages/example-getting-started/README.md

```diff
@@ -0,0 +1,419 @@
+# @loopback/example-getting-started
+
+This is the basic tutorial for getting started with Loopback 4!
+
+**NOTICE**: This tutorial is currently under construction! This notice will be
+removed when it is ready for use!
+
+## Prerequisites
+
+Before we can begin, you'll need to make sure you have some things installed:
+- [Node.js](https://nodejs.org/en/) at v6.x or greater
+- [TypeScript](http://www.typescriptlang.org/)
+- [Git](https://git-scm.com/)
+
+Additionally, this tutorial assumes that you are comfortable with
+certain technologies, languages and concepts.
+- JavaScript (ES6)
+- [npm](https://www.npmjs.com/)
+- [REST](https://en.wikipedia.org/wiki/Representational_state_transfer)
+- [Using Git](https://githowto.com/)
+
+## Setup
+1. Clone this repository if you haven't already:
+```
+git clone https://github.com/strongloop/loopback-next
```

Comment:
Update this to use `lb4 example getting-started` instead of cloning the mono-repo. You should be good after this change.

Reply:
+1

Reply:
IMO, the correct instructions are along the following lines:

 1. `npm install -g loopback4-cli`
 2. `lb4 example getting-started`
 3. `cd loopback4-example-getting-started`
 4. `npm install`
 5. `npm start`